cli/command/cluster/create.go

```diff
@@ -21,33 +24,76 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.IntVarP(&opts.workers, "workers", "w", 2, "Initial number of worker nodes")
-	flags.IntVarP(&opts.managers, "managers", "m", 3, "Intial number of manager nodes")
+	flags.IntVarP(&opts.managers, "managers", "m", 3, "Initial number of manager nodes")
```

Comment:
The options are generic options for the `cluster create` command. The choices I'm considering for changing this are either that CLI users only use options that are specific for each provider (so for example, aws: `--parameter ClusterSize=5`; local: `--parameter workers=5` or else we automatically map these generic options to provider specific options internally (for example for aws we map `--workers=5` to `--parameter ClusterSize=5`). The second options is nice, but a bit magical; the first option is more explicit. I'm leaning toward implementing the first one for right now, but the second option is easy enough to do. We will need to make sure to document and make clear to users that there are standard options that we handle with the command, and then more provider-specific options that are handled in a provider-specific way.